### DISCUSSION TITLE
Ecoharmonogram reads the wrong city (Poland)

### DISCUSSION THREAD

**Discussion by Dzieckozbulerbyn** [mampfes/hacs_waste_collection_schedule#2021] [Q&A]
Hi,

I apologize for my bad English. I'm newbie and I've searched half of the internet but couldn't find an answer; maybe I'm not good at reading :D. 
I'm from Poland, and I wanted to use integration while working with the "ecoharmonogram" application. I have a problem with configuring this integration. 
I live in a city that shares the same name as five other cities listed in the "ecoharmonogram" application. When I enter my address details, the integration retrieves data from a different city than mine. Even after changing the street, the integration still retrieves data from the first city on the "ecoharmonogram" list. If such a street does not exist in that city, the integration shows the status as "off". 
I have tried different identifiers such as "gmina", "gm", "miasto", "m". I have used dots, commas, underscores, hyphens, and more. I also tried adding optional data according to the documentation: additional_sides_matcher: "Zawodzie-Dąbie - Zab. jedn. Zawodzie-Dąbie - Zab. wiel." or "Single-family housing" or "jedn", but it didn't help. 
The integration still retrieves data from the first city listed in the "ecoharmonogram" application. 
Is there a way to enter the province (województwo) somewhere? 
In the integration documentation, my city is entered as one word, but upon entering this name into the application, several other cities with the same name appear. 
How can I deal with this?

-----------------------------------
[PL]
_Przeszukałem pół internetu, lecz nie znalazłem odpowiedzi, może nie umiem dobrze czytać :D.
Jestem z Polski, chciałem użyć integracji przy współpracy z aplikacją "ecoharmonogram".
Mam problem z konfiguracją tej integracji. 
Mieszkam w mieście, które posiada taką samą nazwę jak pięć innych miast widniejących w aplikacji "ecoharmonogram". Gdy wpisuję dane adresowe, integracja pobiera dane z innego miasta niż moje. 
Mimo zmiany ulicy, integracja nadal pobiera dane z pierwszego miasta z listy aplikacji "ecoharmonogram", a jeśli takiej ulicy nie ma w tym mieście, integracja pokazuje status "off".
Próbowałem różnych nazw, począwszy od: "gmina", "gm", "miasto", "m". Używałem kropek, przecinków, podkreśleń, myślników i innych. Próbowałem również dodawać opcjonalne dane według dokumentacji: additional_sides_matcher: "Zawodzie-Dąbie - Zab. jedn. Zawodzie-Dąbie - Zab. wiel." lub "Zabudowa jednorodzinna" lub "jedn", lecz to nie pomogło. 
Integracja nadal pobiera dane z pierwszego miasta które jest na liście w aplikacji "ecoharmonogram". 
Czy jest możliwość wpisania gdzieś województwa?
W dokumentacji integracji moje miasto wpisane jest jako jedno słowo, a po wpisaniu tej nazwy w aplikację, pojawia się kilka innych miast o takiej samej nazwie.
Jak sobie z tym poradzić?_

**Comment by 5ila5** [comment]
This sound like a duplicate of #1963
